libkbfs/folder_branch_ops.go

```diff
@@ -4976,7 +4982,9 @@ func (fbo *folderBranchOps) registerAndWaitForUpdates() {
 					fbo.log.CDebugf(ctx, "Abandoning updates since we can't "+
 						"read the newest metadata: %+v", err)
 					fbo.status.setPermErr(err)
-					cancel()
+					// No need to lock here, since `cancelUpdates` is
+					// only set within this same goroutine.
```

Comment:
`ClearPrivateFolderMD` doesn't write to the `fbo.cancelUpdates` field, so whatever's there is safe to read from this goroutine without taking the lock.  Furthermore, `ClearPrivateFolderMD` waits until the goroutine completes before releasing any of its locks, so no new goroutine can be spawned to update it either.  Any suggestions how I can make that more clear?